tests/unit/lms/views/error_test.py

```diff
@@ -1,167 +1,155 @@
 from unittest import mock
```

Comment:
This file looks like total spaghetti in the diff, just open the new one and it's quite obvious what's been done